app/views/items/_item.html.erb

```diff
@@ -1,5 +1,6 @@
 <% @items.each do |item| %>
   <ul>
-    <li><%= item.name %></li>
+    <li><%= link_to "", @item, method: :delete, class: 'gylphicon glyphicon-ok' %>
```

Comment:
ok, let me come back to the code here.  It looks to me like you're looping through some collection of items called `@items`.  At least that's what I'm assuming is happening without looking at anything other than the few lines of code shown here.
So I'm assuming you're wanting to display a link for each item in the collection.  What variable do you use to represent each item in the collection?